[2025-06-09T07:31:26.981+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T07:31:27.017+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:31:24.413956+00:00 [queued]>
[2025-06-09T07:31:27.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:31:24.413956+00:00 [queued]>
[2025-06-09T07:31:27.029+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-09T07:31:27.045+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_from_netsuite> on 2025-06-09 07:31:24.413956+00:00
[2025-06-09T07:31:27.053+0000] {standard_task_runner.py:63} INFO - Started process 38 to run task
[2025-06-09T07:31:27.056+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'netsuite_saved_search_4532_memory', 'extract_from_netsuite', 'manual__2025-06-09T07:31:24.413956+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/netsuite_4532_dag.py', '--cfg-path', '/tmp/tmpyot5grjo']
[2025-06-09T07:31:27.059+0000] {standard_task_runner.py:91} INFO - Job 361: Subtask extract_from_netsuite
[2025-06-09T07:31:27.145+0000] {task_command.py:426} INFO - Running <TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:31:24.413956+00:00 [running]> on host 5885a648a6f1
[2025-06-09T07:31:27.337+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='netsuite_saved_search_4532_memory' AIRFLOW_CTX_TASK_ID='extract_from_netsuite' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T07:31:24.413956+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T07:31:24.413956+00:00'
[2025-06-09T07:31:27.342+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T07:31:28.999+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T07:31:29.000+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/netsuite_4532_dag.py", line 51, in extract_from_netsuite
    raise Exception("❌ Đăng nhập NetSuite thất bại hoặc trả về HTML")
Exception: ❌ Đăng nhập NetSuite thất bại hoặc trả về HTML
[2025-06-09T07:31:29.054+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=netsuite_saved_search_4532_memory, task_id=extract_from_netsuite, run_id=manual__2025-06-09T07:31:24.413956+00:00, execution_date=20250609T073124, start_date=20250609T073127, end_date=20250609T073129
[2025-06-09T07:31:29.080+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 361 for task extract_from_netsuite (❌ Đăng nhập NetSuite thất bại hoặc trả về HTML; 38)
[2025-06-09T07:31:29.098+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-09T07:31:29.133+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T07:31:29.136+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
